src/components/SongEntry/SongEntry.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components/macro'
+
+interface SongEntryProps {
+  name: string
+  artists: SpotifyApi.ArtistObjectSimplified[]
+}
+
+function SongEntry({ name, artists }: SongEntryProps): JSX.Element {
+  const allArtists: string = artists.map(artist => artist.name).join(', ')
+
+  return (
+    <Container>
+      <SongName>{name}</SongName>
+      <ArtistName>{allArtists}</ArtistName>
+    </Container>
+  )
+}
+
+export default SongEntry
+
+const Container = styled.li`
```

Comment:
I prefer to not use `li` as component wrapper elements, but `section`. If you want to use it as an `li`, you could do it with `<SongEntry as="li" />`